util/chplenv/defaults.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+"""
+Check .chplconfig file and environment variables for default value overrides
+"""
+from __future__ import print_function
+import os
+import sys
+from sys import stderr
```

Comment:
You don't actually use stderr very much, so I have a slight preference of just calling `sys.stderr` and removing this import. I can be talked out of that if you care though